test/debugger.ts

```diff
@@ -109,10 +109,7 @@ export class Debugger extends common.ServiceObject {
    *     breakpoints set by all users, or just by the caller (default false).
    * @param {boolean=} options.includeInactive - Whether or not to include
    *     inactive breakpoints in the list (default false).
-   * @param {boolean=} options.stripResults - If set to true, breakpoints will be
```

Comment:
In that case, I think it would probably be better to remove this parameter in a separate PR and describe in the PR description that the param is being removed because it is deprecated.